### PR TITLE
Update mergify

### PR DESCRIPTION
**Pull request by jeffoverflow**
Signed-off-by: quicksilver <zhifeng.zhang@zilliz.com>

### CODE REVIEW

**Review by wxyucs — APPROVED** [review]
/lgtm
/approve
/kind improvement

**Review by XuanYang-cn — APPROVED** [review]
/lgtm

**Review by wxyucs — CHANGES_REQUESTED** [review]

File: .github/mergify.yml [hunk 1]
```diff
@@ -30,6 +30,15 @@ pull_request_rules:
         add:
           - ci-passed
 
+  - name: Test passed for mergify changed
+    conditions:
+      - base=master
+      - files=.github/mergify.yml
```
[the remark this answers is not in the document]
Comment (jeffoverflow) [reply .github/mergify.yml]: Fixed

**Review by wxyucs — APPROVED** [review]
/lgtm
/approve
/unhold

**Pull request merged**